Riot/Modules/Room/VoiceMessages/VoiceMessageAudioPlayer.swift

```diff
@@ -198,13 +199,23 @@ class VoiceMessageAudioPlayer: NSObject {
                 (delegate as? VoiceMessageAudioPlayerDelegate)?.audioPlayerDidFinishPlaying(self)
             }
         }
+        
+        appBackgroundObserver = NotificationCenter.default.addObserver(forName: UIApplication.didEnterBackgroundNotification, object: nil, queue: nil) { [weak self] _ in
+            guard let self = self, !BuildSettings.allowBackgroundAudioMessagePlayback else { return }
```

Comment:
Would it be worth adding a check for `isPlaying` in here too?

Reply:
I don't think so. Pausing a paused stream looks like a nop.

Reply:
Even for any of the delegates that get called?

Reply:
Tried it, I'm not seeing anything bad in the app.